python/ray/dag/tests/experimental/test_accelerated_dag.py

```diff
@@ -136,18 +136,47 @@ def test_basic(ray_start_regular):
     compiled_dag.teardown()
 
 
-def test_multiple_returns_not_supported(ray_start_regular):
+def test_two_returns_first(ray_start_regular):
+    a = Actor.remote(0)
+    with InputNode() as i:
+        o1, o2 = a.return_two.bind(i)
+        dag = MultiOutputNode([o1])
```

Comment:
```suggestion
        dag = o1
```

(make sure the simpler version of the test works)

Reply:
Done.